client/src/components/MinMaxSelect.tsx

```diff
@@ -1,89 +1,193 @@
-import React from "react";
-import { Trans } from "@lingui/macro";
+import React, { InputHTMLAttributes } from "react";
+import { Trans, t } from "@lingui/macro";
+import { i18n } from "@lingui/core";
 import classnames from "classnames";
 import helpers from "util/helpers";
 import { Alert } from "./Alert";
-import { FilterNumberRange, MINMAX_DEFAULT } from "./PropertiesList";
+import { FilterNumberRange, NUMBER_RANGE_DEFAULT } from "./PropertiesList";
+
+type MinMaxErrors = { min: boolean; max: boolean; msg: JSX.Element | undefined };
```

Comment:
nit: I think the current variable names get harder to read when we start referencing the values like `MinMaxErrors.min || MinMaxErrors.max`. but this is also personal preference so truly take it or leave it!

```suggestion
type RangeValidation = { isMinValid: boolean; isMaxValid: boolean; errorMsg: JSX.Element | undefined };
```

Reply:
oh yeah great point! Just updated for this change - thanks!